src/Plugin/EntityBrowser/Widget/View.php

```diff
@@ -209,4 +210,41 @@ public function submit(array &$element, array &$form, FormStateInterface $form_s
     $this->selectEntities($entities, $form_state);
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  public function buildConfigurationForm(array $form, FormStateInterface $form_state) {
+    $options = [];
+    // Get only those enabled Views that have entity_browser displays.
+    $displays = Views::getApplicableViews('entity_browser_display');
+    foreach ($displays as $display) {
+      list($view_id, $display_id) = $display;
+      $view = $this->entityManager->getStorage('view')->load($view_id);
+      $options[$view_id . '.' . $display_id] = $this->t('@view : @display', array('@view' => $view->label(), '@display' => $view->get('display')[$display_id]['display_title']));
+    }
```

Comment:
Well, I think we should think about two things.

1. Should we introduce a dependency on views.module in entity_browser.info.yml? Currently we don't have this, and we don't even check if Views is enabled, so one could enable Entity Browser without Views, and will end up having nothing but a cryptic error message saying entity type 'view' is not found, if at all. (This error message was already present before this PR.)

2. If we decide not to introduce a hard dependency on views.module (see above), then what should be the most appropriate place to check for it?

In other words: is there any real use case for Entity Browser without Views?